### PR TITLE
Nutmeg.2 Release

### PR DESCRIPTION
**Pull request by BbrSofiane**
- [Feature] Upgrade all applications to open-release/nutmeg.2.

### CODE REVIEW

**Review by regisb — APPROVED** [review]
Lgtm! Is this still a draft?

**Comment by BbrSofiane** [comment]
I've made it ready for review now. 

I'm still building the image to do a quick check.

**Comment by BbrSofiane** [comment]
I'm getting this error:
```
pymongo.errors.ServerSelectionTimeoutError: client is configured to connect to a replica set named '' but this node belongs to a set named 'None', Timeout: 30s, Topology Description: <TopologyDescription id: 62f34e196f500a0644fc30ee, topology_type: Single, servers: [<ServerDescription ('mongodb', 27017) server_type: Unknown, rtt: None, error=ConfigurationError("client is configured to connect to a replica set named '' but this node belongs to a set named 'None'")>]>
```

I'll look into later today.

**Comment by insad** [comment]
Something to do with https://github.com/openedx/edx-platform/commit/46888e874d655d1076be04d68a42c621faae8ad6 ?

**Comment by regisb** [comment]
I've got a bad feeling about this... I know that @kdmccormick pushed a fix for this change in the nightly branch but I can't find it now. If you are working on this, please make sure that the new tag is working for the MFE plugin as well. If you are sufficiently confident in the new tag, please ping me and send me an email.

**Comment by BbrSofiane** [comment]
Thanks, @insad! I added these settings and it fixed the issue. 

I'm not sure what impact it would have for someone using replica sets. Who would be a  good person to ask?

**Comment by insad** [comment]
Not using replica sets, but will try immediately this release.

**Comment by kdmccormick** [comment]
@BbrSofiane , I think @regisb is thinking of @ormsbee 's pymongo fix here? https://github.com/overhangio/tutor/pull/699

**Comment by BbrSofiane** [comment]
Done! This is ready to be merged.

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Every user-facing change should have an entry in this changelog. Please respect
 
 ## Unreleased
 
+## v14.0.5 (2022-08-09)
```
Comment (fghaas) [inline CHANGELOG.md:21]: @BbrSofiane Looks like the date needs to be updated here? (And presumably in the commit message, too?)

Comment (regisb) [reply CHANGELOG.md:21]: I modified the date and merged the PR.

**Comment by regisb** [comment]
I changed the date as suggested above and then took the liberty to cherry-pick the commits. Thus I'm closing this now. I release will follow very shortly.

**Pull request closed without merging**